lib/payload/spawn-monitor.vala

```diff
@@ -38,7 +38,11 @@ namespace Frida {
 			var interceptor = Gum.Interceptor.obtain ();
 
 #if WINDOWS
-			interceptor.attach_listener (Gum.Module.find_export_by_name ("kernelbase.dll", "CreateProcessInternalW"), this);
+			var cpiw_export = Gum.Module.find_export_by_name ("kernelbase.dll", "CreateProcessInternalW");
```

Comment:
Let's name it `cpiw` or `create_process_internal` for consistency with the Darwin code below.